chapters/intro/main.tex

```diff
@@ -35,33 +35,66 @@ \chapter{Introduction}
 presented in this thesis, as well as signposting the software projects
 themselves.
 
 
-\section{Thesis structure}\label{sec:thesis}
+\section{Thesis objective}\label{sec:objective}
+
+The primary purpose of this thesis is to utilise machine learning to better
+understand a healthcare population. This purpose is achieved through the novel
+methods for clustering initialisation and algorithm evaluation presented herein.
+The co-sponsors of this project, NHS Wales Cwm Taf Morgannwg University
+Health Board (UHB), are seeking to reveal new insights into their patient
+population through the use of machine learning. In the context of machine
+learning, the term `utilisation' typically refers to the application of some
+existing machine learning apparatus to a dataset. This thesis considers a more
+nuanced definition. Here, `utilisation' is considered as the culmination of
+three parts: \emph{creation}, \emph{evaluation}, and \emph{application}. Each of
+these components is essential to properly utilising machine learning, and the
+process is comparable to honing a craft.
+
+First, a tool is fashioned for a particular purpose, as a machine learning
+method would be for a problem --- or class thereof. Once a prototype has been
+created, the tool must be evaluated and adjusted. This process improves both the
+skilled use of the tool and the tool itself. Evaluating a machine
+learning method extensively is analogous to this process. With a refined tool
+and skilled hand, expertise in that craft can be demonstrated by applying it for
+its intended purpose; this is the final (and often only) stage when utilising
+machine learning in the real world.
+
+\ctmuhb\ are particularly concerned with understanding the operational
+characteristics of their patients presenting chronic obstructive pulmonary
+disease (COPD). COPD is a respiratory condition with known links to deprivation,
+and often presents as a comorbidity, i.e. in concurrence with at least one other
+condition. These affiliations make living with and treating COPD inherently
+difficult, emphasising the importance of studying it closely.
+
+This thesis considers how clustering can be utilised to better understand the
+COPD population treated by \ctmuhb. In order to carry out this utilisation, this
+thesis incorporates three themes from modern OR: algorithm evaluation,
+clustering and segmentation, and operational healthcare modelling. These
+seemingly disparate themes are presented as a triptych, where each theme
+corresponds to its own chapter. With each themed chapter, there is a clear
+contribution to a component of utilising machine learning.
+
+The broader issue of algorithm evaluation is addressed Chapter~\ref{chp:edo},
+where a novel framework for understanding the objective quality of an algorithm
+is presented. In Chapter~\ref{chp:kmodes}, an extension to an existing
+clustering algorithm is devised and subsequently evaluated using the framework
```

Comment:
It took me a little while to put my finger on what I think is niggling me in this paragraph.

You say in the previous paragraph:

>  With each themed chapter, there is a clear contribution to a component of utilising machine learning.
 
So I was expecting to see the next three chapters be directly corresponding to creation, evaluation and application.

In reality `chp:edo` and `chp:kmodes` do both "creation" and "evaluation" don't they? 

This of course isn't a problem, just the narrative doesn't flow as well.

Reply:
Fixed by 12904ef